### ISSUE TITLE
markup decorative hr tags as decorative

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4212]
`<hr>` has been used for visual formatting, but it is focusable and read as 'separator, dimmed' which is confusing.

update the markup so that these are not focusable and are not described by screen readers.

examples:
<img width="270" alt="image" src="https://github.com/BirkbeckCTP/janeway/assets/5330770/c1d62c43-6bb4-4df4-a5e5-2a48fa3a27b1">
The almost invisible pale line under 'Research Article' is an `<hr>`.

**Comment by StephDriver** [comment]
from #4194 

- Sample 5.1, item 7. the line under this seems to be decorative, but is read out as 'separator, dimmed'
- Sample 5.3, item 3. "separator, dimmed" the `<hr>` following the about and before news is read aloud, but this is decorative.

**Comment by ajrbyers** [comment]
I think we will need to assess the use of hr tags. They are legitimately separators used the breakup content so no technically “decorative”.

**Comment by StephDriver** [comment]
It was the 'dimmed' part that felt confusing in a 'why do I need to know this' way.  Perhaps the dimming can be done through the CSS instead?

Further, I think the HR as a semantic element is only when used to separate paragraphs. In the above example I think it is underlying a heading, which is decorative.

see [<hr>: The Thematic Break on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr)

For now, I think a review of HR use would make sense if combined with 
- #4232

**Comment by StephDriver** [comment]
closing because it is now part of #4232
